chronograf/1.10/alpine/Dockerfile

```diff
@@ -25,7 +25,11 @@ RUN set -ex && \
     cp -a /usr/src/chronograf-*/* /usr/bin/ && \
     gpgconf --kill all && \
     rm -rf *.tar.gz* /usr/src /root/.gnupg && \
-    apk del .build-deps
+    apk del .build-deps && \
+    addgroup -S chronograf && \
+    adduser -S chronograf -G chronograf && \
+    mkdir -p /var/lib/chronograf && \
+    chown -R chronograf:chronograf /var/lib/chronograf
```

Comment:
Since you just created the directory, the `-R` is not needed (it is empty).

Reply:
Fixed